bin/changelog/config.js

```diff
@@ -1,23 +1,52 @@
 'use strict';
 
 const pkg = require( '../../package.json' );
+const Config = require( 'merge-config' );
+
+const config = new Config();
+
+const changelogSrcTypes = {
+	MILESTONE: 'MILESTONE',
+	ZENHUB: 'ZENHUB_RELEASE',
+};
 
 const REPO = pkg.repository.url
 	// remove https://github.com:
 	.split( ':' )[ 2 ]
 	// remove the .git ending.
 	.slice( 0, -4 );
```

Comment:
Thinking of changing this to something more robust, since it will only work if your repo name is like this:

```
"repository": {
	"type": "git",
	"url": "https://github.com:woocommerce/woocommerce-gutenberg-products-block.git"
}
``` 

Gutenberg, for example, has this

https://github.com/WordPress/gutenberg/blob/c49b051f3340cccdc59d140b44b450b87239cb01/package.json#L13

might as well add that option to the changelog config

Reply:
done.